### DISCUSSION TITLE
Parameter to skip uninstall action after tests are completed.

### DISCUSSION THREAD

**Discussion by nemanjavukojebmw** [Banno/Gordon#131] [Ideas]
Inside GordonTest Task, after all tests are done, task will uninstall the application package and instrumentation package. This would also remove the data stored in the app storage during the tests.

It would be nice to have possibility of keeping the application data after gordon tests are done, as some reports or screenshots could also be stored there.  

Something similar is introduced in Android Gradle Plugin 8.2 in form of the gradle property for android tests:
`android.injected.androidTest.leaveApksInstalledAfterRun = true`
https://issuetracker.google.com/issues/295039976?hl=ja

Same property could be read by Gordon task, or separate parameter could be added to the task with the same behaviour.

**Comment by joshschriever** [comment]
Yeah, this is a good idea. Shouldn't be too difficult, so this would a great opportunity for a first-time contributor.

**Comment by christianbrennstuhlbmw** [comment]
We will create a PR for this soon - we already got it working locally!

**Comment by christianbrennstuhlbmw** [comment]
Better late than never ;)
https://github.com/Banno/Gordon/pull/134

**Comment by joshschriever** [comment]
Released in [v1.12.1](https://github.com/Banno/Gordon/releases/tag/1.12.1)
